com.basemark.BasemarkGPU.desktop

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env xdg-open
```

Comment:
The first line here is not needed.

Reply:
Resolved in 8648d7cb12eb38ecb026115bd10601a5d2a8536b